datasets/ag_news/README.md

```diff
@@ -0,0 +1,147 @@
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## [Dataset Description](#dataset-description)
+
+- **Homepage:** [http://groups.di.unipi.it/~gulli/AG_corpus_of_news_articles.html](http://groups.di.unipi.it/~gulli/AG_corpus_of_news_articles.html)
+- **Repository:** [More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
+- **Paper:** [More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
+- **Point of Contact:** [More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
+- **Download Size:** 29.88 MB
+
+### [Dataset Summary](#dataset-summary)
+
+AG is a collection of more than 1 million news articles. News articles have been
+gathered from more than 2000 news sources by ComeToMyHead in more than 1 year of
+activity. ComeToMyHead is an academic news search engine which has been running
+since July, 2004. The dataset is provided by the academic comunity for research
+purposes in data mining (clustering, classification, etc), information retrieval
+(ranking, search, etc), xml, data compression, data streaming, and any other
+non-commercial activity. For more information, please refer to the link
+http://www.di.unipi.it/~gulli/AG_corpus_of_news_articles.html .
+
+The AG's news topic classification dataset is constructed by Xiang Zhang
+(xiang.zhang@nyu.edu) from the dataset above. It is used as a text
+classification benchmark in the following paper: Xiang Zhang, Junbo Zhao, Yann
+LeCun. Character-level Convolutional Networks for Text Classification. Advances
+in Neural Information Processing Systems 28 (NIPS 2015).
+
+### [Supported Tasks](#supported-tasks)
+
+[More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
+
+### [Languages](#languages)
+
+[More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
+
+## [Dataset Structure](#dataset-structure)
+
+We show detailed information for up to 5 configurations of the dataset.
+
+### [Data Instances](#data-instances)
+
+#### default
+
+Download size: 29.88 MB
+
+An example of 'train' looks as follows.
+```
+{
+    "label": 3,
+    "text": "New iPad released Just like every other September, this one is no different. Apple is planning to release a bigger, heavier, fatter iPad that..."
+}
+```
+
+### [Data Fields](#data-fields)
+
+The data fields are the same among all splits.
+
+#### default
+- `text`: a `string` feature.
+- `label`: a classification label, with possible values including `World`, `Sports`, `Business`, `Sci/Tech`.
```

Comment:
Would it be possible to make the correspondence between label names and ids explicit ? In particular since the actual data that are stored are the label ids as integers.
```
- `label`: a classification label that corresponds to one the following labels: `World` (0), `Sports` (1), `Business` (2), or `Sci/Tech` (3).
```

Reply:
yup, doing that!